src/lib/pages/publish-module/components/Footer.tsx

```diff
@@ -10,29 +10,28 @@ interface FooterProps {
 
 export const Footer = ({ isLoading = false, fieldAmount }: FooterProps) => {
   const router = useRouter();
+  const publishText =
+    fieldAmount > 1 ? `Publish ${fieldAmount} Modules` : "Publish Module";
   return (
     <Grid
       w="full"
       px={12}
       bg="gray.900"
       h="70px"
-      bottom="0"
+      bottom={0}
       position="sticky"
       zIndex={2}
-      templateAreas={`"prespace main postspace"`}
-      templateColumns="1fr 5fr 4fr"
+      templateAreas={`"prespace main accordion postspace"`}
```

Comment:
templateAreas="prespace main accordion postspace"

Reply:
the meaning is different css-wise